web/src/widget/text_input.rs

```diff
@@ -50,15 +54,19 @@ impl<'a, Message> TextInput<'a, Message> {
     ///
     /// It expects:
     /// - some [`State`]
-    /// - a placeholder
+    /// - an optional placeholder
+    /// - a label describing the input
+    /// - an id
```

Comment:
placeholders aren't replacements for labels. i made the placeholder optional so that people can decide whether they want one or not, but made label text required